docs/sources/mimir/release-notes/v2.15.md

```diff
@@ -0,0 +1,128 @@
+
+mimirtool's analyze ruler/prometheus commands now support bearer tokens.
+
+HTTP client settings can now be tuned for GCS and Azure backends via an `http` block or corresponding CLI flags.
+
+The compactor now refreshes deletion marks concurrently when updating the bucket index.
+
+The number of Memcached replicas for each type of cache can now be set using  configuration settings when using jsonnet.
+
+## Important changes
+
+In Grafana Mimir 2.15, the following behavior has changed:
+
+The following alertmanager metrics are not exported for a `user` when the metric value is zero:
+
+- `cortex_alertmanager_alerts_received_total`
+- `cortex_alertmanager_alerts_invalid_total`
+- `cortex_alertmanager_partial_state_merges_total`
+- `cortex_alertmanager_partial_state_merges_failed_total`
+- `cortex_alertmanager_state_replication_total`
+- `cortex_alertmanager_state_replication_failed_total`
+- `cortex_alertmanager_alerts`
+- `cortex_alertmanager_silences`
+
+PromQL compatibility has been upgraded from Prometheus 2.0 to 3.0. More details can be found in the [Prometheus documentation](https://prometheus.io/docs/prometheus/3.0/migration/#promql). The following changes are of note:
+
+- The `.` pattern in regular expressions in PromQL now matches newline characters.
+- Lookback and range selectors are left open and right closed (previously left closed and right closed).
+- Native histograms now use exponential interpolation.
+
+Backwards compatibility in dashboards and alerts for `thanos_memcached_`-prefixed metrics has been removed. These metrics were removed in 2.12 in favor of `thanos_cache_`-prefixed metrics.
+
+Support for Redis as a cache backend has been removed from jsonnet.
+
+The following deprecated configuration options were removed in this release:
+
+- `-distributor.direct-otlp-translation-enabled`, which has been enabled by default since 2.13 and is now considered stable.
+- `-query-scheduler.prioritize-query-components`, which is always enabled now
+- `-ingest-storage.kafka.ongoing-records-per-fetch` and `-ingest-storage.kafka.startup-records-per-fetch`, which have been removed in favour of `-ingest-storage.kafka.max-buffered-bytes`
+
+## Experimental features
+
+Grafana Mimir 2.15 includes some features that are experimental and disabled by default.
+Use these features with caution and report any issues that you encounter:
+
+A streaming PromQL engine can now be enabled with `-querier.query-engine=mimir`.
+
+The query-frontend can now cache non-transient errors using the experimental flags `-query-frontend.cache-errors` and `-query-frontend.results-cache-ttl-for-errors`.
+
+The query-frontend and querier both support an experimental PromQL function, `double_exponential_smoothing`, which can be enabled by setting `-querier.promql-experimental-functions-enabled=true` or `-query-frontend.promql-experimental-functions-enabled=true`.
```

Comment:
[nit] The "or" makes it sound like a user can set either of these and they'll have the same effect, but my understanding is that both need to be set for this to work properly.

```suggestion
The query-frontend and querier both support an experimental PromQL function, `double_exponential_smoothing`, which can be enabled by setting `-querier.promql-experimental-functions-enabled=true` and `-query-frontend.promql-experimental-functions-enabled=true`.
```